### ISSUE TITLE
Unexpected quantize and dequantize behaviors.

### ISSUE THREAD

**Issue by zaneghost** [NVIDIA/TensorRT#1362]
## Description

I am trying to use quantize/dequantize layer using python api :
    network.add_quantize
    network.add_dequantize
However, the scaleNode getChannelAxis error keeps coming out, blocks further operation

```
[TensorRT] ERROR: 2: [scaleNode.cpp::getChannelAxis::20] Error Code 2: Internal Error ((Unnamed Layer* 2) [Quantize]: unexpected negative axis)
```

P.S. Also, adding quantize layer changed input tensor shape behavior
```
-------------------- Shape of: conv_1_input_scale (1,)
-------------------- Shape of: inputs (1, 3, 256, 256)
[TensorRT] ERROR: 2: [scaleNode.cpp::getChannelAxis::20] Error Code 2: Internal Error ((Unnamed Layer* 2) [Quantize]: unexpected negative axis)
-------------------- Shape of: conv_1_input_scale (0)
[TensorRT] ERROR: 2: [scaleNode.cpp::getChannelAxis::20] Error Code 2: Internal Error ((Unnamed Layer* 2) [Quantize]: unexpected negative axis)
-------------------- Shape of: inputs (0)
```

P.S.S I wander why constant layer can directly call shape?
```
conv_1_input_scale = network.add_constant([1], np.array(1/127, np.float32))
print("-------------------- Shape of: conv_1_input_scale", conv_1_input_scale.shape)
```

## Environment

**TensorRT Version**: 8.0.1.6
**NVIDIA GPU**: TESLA T4
**NVIDIA Driver Version**: 465.31
**CUDA Version**: V11.3.109
**CUDNN Version**: 8.2.1
**Operating System**: ubuntu 1604
**Python Version (if applicable)**: 3.5.2
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Steps To Reproduce

```
import sys, os
import numpy as np
import tensorrt as trt
from time import perf_counter

import common_ori as common


TRT_LOGGER = trt.Logger(trt.Logger.WARNING)

ROOT = "./fix_params/"

class ModelData(object):
    INPUT_NAME = "input_image"
    INPUT_SHAPE = (1,256,256,3)
    # +++++++++++++++++++++++++++++++++++++++++
    OUTPUT_NAME_0 = "output_feature"
    DTYPE = trt.float32

def populate_network(network):
    '''
    # inputs
    '''
    input_tensor = network.add_input(name=ModelData.INPUT_NAME, dtype=ModelData.DTYPE, shape=ModelData.INPUT_SHAPE)

    inputs = network.add_shuffle(input_tensor)
    inputs.first_transpose = trt.Permutation((0,3,1,2))

    '''
    # conv_1
    '''
    conv_1_input_scale = network.add_constant([1], np.array(1/127, np.float32))
    print("-------------------- Shape of: conv_1_input_scale", conv_1_input_scale.get_output(0).shape)
    print("-------------------- Shape of: inputs", inputs.get_output(0).shape)
    buf_1 = network.add_quantize(input = inputs.get_output(0), scale = conv_1_input_scale.get_output(0))
    print("-------------------- Shape of: conv_1_input_scale", conv_1_input_scale.get_output(0).shape)
    print("-------------------- Shape of: inputs", inputs.get_output(0).shape)
    # print("-------------------- Shape of: buf_1", buf_1.get_output(0).shape)
    print('1')
    # inputs_int8 = network.add_dequantize(input=buf_1.get_output(0), scale = conv_1_input_scale.get_output(0))
    print('2')

    conv_1_weight_scale = network.add_constant([1], np.array(0.8419753909111023/127, np.float32))
    conv_1_weight_float = network.add_constant([12,3,2,2], np.load(ROOT+'simple_mse_analysis_anfi_quant_conv_1_kernel_read.npy'))
    buf_2 = network.add_quantize(input = conv_1_weight_float.get_output(0), scale = conv_1_weight_scale.get_output(0))
    conv_1_weight = network.add_dequantize(input=buf_2.get_output(0), scale = conv_1_weight_scale.get_output(0))

    conv_1 = network.add_convolution(input=inputs_int8.get_output(0), num_output_maps=12, kernel_shape=(2, 2), 
        kernel=conv_1_weight, 
        bias=np.load(ROOT+'simple_mse_analysis_anfi_quant_conv_1_bias_read.npy'))
    conv_1.stride = (2, 2)
    conv_1.get_output(0).name = ModelData.OUTPUT_NAME_0
    network.mark_output(tensor=conv_1.get_output(0))

use_int8 = 1

def build_engine():
    EXPLICIT_BATCH = 1 << (int)(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH)
    # network_creation_flag = 1 << int(trt.NetworkDefinitionCreationFlag.EXPLICIT_PRECISION)
    flag = EXPLICIT_BATCH# | network_creation_flag
    with trt.Builder(TRT_LOGGER) as builder, builder.create_network(flag) as network, builder.create_builder_config() as config:        
        config.max_workspace_size = common.GiB(10)
        config.set_flag(trt.BuilderFlag.FP16)
        if use_int8:
            config.set_flag(trt.BuilderFlag.INT8)

        populate_network(network)
        return builder.build_engine(network, config)

def main():
    # Do inference with TensorRT.
    with build_engine() as engine:
				
        with engine.create_execution_context() as context: 
            if use_int8:
                engine_name = "test_long_int8"
            else:
                engine_name = "test_long_fp32"
            with open('./%s.engine'%engine_name, "wb") as f:
                f.write(engine.serialize())              

if __name__ == '__main__':
    main()

```

**Comment by jerryzh168** [comment]
I think maybe you need to set the axis for quantize/dequantize layer to be 0, I did similar things recently: https://github.com/pytorch/pytorch/pull/60589

**Comment by zaneghost** [comment]
> I think maybe you need to set the axis for quantize/dequantize layer to be 0, I did similar things recently: [pytorch/pytorch#60589](https://github.com/pytorch/pytorch/pull/60589)

thanks, and thats correct!
